common/containers/Tabs/SendTransaction/components/ConfirmationModal.tsx

```diff
@@ -101,84 +101,87 @@ class ConfirmationModal extends React.Component<Props, State> {
-                    <Identicon size="100%" address={toAddress} />
-                  </div>
-                </div>
 
-                <ul className="ConfModal-details">
-                  <li className="ConfModal-details-detail">
-                    You are sending from <code>{from}</code>
-                  </li>
-                  <li className="ConfModal-details-detail">
-                    You are sending to <code>{toAddress}</code>
-                  </li>
-                  <li className="ConfModal-details-detail">
-                    You are sending with a nonce of <code>{nonce}</code>
-                  </li>
-                  <li className="ConfModal-details-detail">
-                    You are sending{' '}
-                    <strong>
-                      {value} {symbol}
-                    </strong>{' '}
-                    with a gas price of <strong>{gasPrice} gwei</strong>
-                  </li>
-                  <li className="ConfModal-details-detail">
-                    You are interacting with the <strong>{node.network}</strong>{' '}
-                    network provided by <strong>{node.service}</strong>
-                  </li>
-                  {!token && (
+                  <ul className="ConfModal-details">
                     <li className="ConfModal-details-detail">
-                      {data ? (
-                        <span>
-                          You are sending the following data:{' '}
-                          <textarea
-                            className="form-control"
-                            value={data}
-                            rows={3}
-                            disabled={true}
-                          />
-                        </span>
-                      ) : (
-                        'There is no data attached to this transaction'
-                      )}
+                      You are sending from <code>{from}</code>
                     </li>
-                  )}
-                </ul>
-
-                <div className="ConfModal-confirm">
-                  {translate('SENDModal_Content_3')}
+                    <li className="ConfModal-details-detail">
+                      You are sending to <code>{toAddress}</code>
+                    </li>
+                    <li className="ConfModal-details-detail">
+                      You are sending with a nonce of <code>{nonce}</code>
+                    </li>
+                    <li className="ConfModal-details-detail">
+                      You are sending{' '}
+                      <strong>
+                        {value} {symbol}
+                      </strong>{' '}
+                      with a gas price of <strong>{gasPrice} gwei</strong>
+                    </li>
+                    <li className="ConfModal-details-detail">
+                      You are interacting with the
+                      <strong>{node.network}</strong>{' '}
```

Comment:
`{' '}` before `{node.network}` to address whitespace issue in screenshot

Reply:
Good catch!